File: core/CreateTournament/Models/PlayerStats.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace CreateTournament.Models
{
    public class PlayerStats: DeleteableEntity
    {
        public int YellowCard {  get; set; }
        public int RedCard { get; set; }
        public int Score { get; set; }
        public int Assits { get; set; }
        public int PlayerId { get; set; }
        [ForeignKey("PlayerId")]
        public Player Player { get; set; }
        public int MatchResultId { get; set; }
        [ForeignKey("MatchResultId")]
        public MatchResult MatchResult { get; set; }
    }
}

File: client/src/app/core/models/classes/PlayerStats.ts
import { IPlayerStats } from "../interfaces/IPlayerStats";

export class PlayerStats implements IPlayerStats {
	id: number;
	yellowCard: number;
	redCard: number;
	score: number;
	assists: number;
	playerId: number;
	matchResultId: number;

	constructor(playerId: number, matchResultId: number) {
		this.yellowCard = 0;
		this.redCard = 0;
		this.score = 0;
		this.assists = 0;
		this.playerId = playerId;
		this.matchResultId = matchResultId;
	}
}
